publication-commons/src/main/java/no/unit/nva/publication/model/business/TicketEntry.java

```diff
@@ -127,6 +127,7 @@ public static void setServiceControlledFields(TicketEntry ticketEntry,
         ticketEntry.setCreatedDate(now);
         ticketEntry.setModifiedDate(now);
         ticketEntry.setIdentifier(identifierProvider.get());
+        ticketEntry.setFinalizedDate(nonNull(ticketEntry.getFinalizedDate()) ? now : null);
```

Comment:
Is it okay to handle it here? Instead of creating identical service methods for persisting completed ticket with only difference that finalized data is set by default.

Reply:
Try it; there might be a few edge-cases we cannot see right now.